src/engine/strat_engine/blockdev.rs

```diff
@@ -2,6 +2,9 @@
 // License, v. 2.0. If a copy of the MPL was not distributed with this
 // file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
+use devicemapper::Device;
```

Comment:
These should go with the other external non standard dependencies, around line 20.